app/templates/events/view.hbs

```diff
@@ -1,6 +1,59 @@
 {{#unless (includes session.currentRouteName 'events.view.edit')}}
-  {{!-- Tabs for the event view page will come here --}}
-  {{outlet}}
+  <div class="ui grid stackable">
+    <div class="row">
+      <div class="four wide column">
+        <h2 class="ui header">Fossasia Test</h2>
+      </div>
+      <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
+        {{#if device.isMobile}}
+          <div class="ui icon fluid buttons">
+            <button class="ui button"><i class="unhide icon"></i></button>
+            <button class="ui button "><i class="check icon"></i></button>
+            <button class="ui button "><i class="copy icon"></i></button>
+            <button class="ui red button icon"><i class="trash icon"></i></button>
```

Comment:
The icon class can be removed here. Since it's already added to wrapping buttons div